app/frontend/src/features/connections/friends/FriendSummaryView.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+import { Link } from "react-router-dom";
+import TextBody from "../../../components/TextBody";
+import { User } from "../../../pb/api_pb";
+
+const useStyles = makeStyles((theme) => ({
+  friendItem: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    padding: `0 ${theme.spacing(1)}`,
+  },
+  friendLink: {
+    color: theme.palette.text.primary,
+    textDecoration: "none",
+  },
+  userLoadErrorAlert: {
+    borderRadius: 0,
+  },
+}));
+
+interface FriendSummaryViewProps {
+  children?: React.ReactNode;
+  friend?: User.AsObject;
+}
+
+function FriendSummaryView({ children, friend }: FriendSummaryViewProps) {
+  const classes = useStyles();
+
+  return friend ? (
+    <Box className={classes.friendItem} key={friend.userId}>
+      <Link className={classes.friendLink} to={`/user/${friend.username}`}>
+        <Typography variant="h2" component="h3">
+          {friend.name}
+        </Typography>
+        <TextBody>@{friend.username}</TextBody>
+      </Link>
+      {children}
+    </Box>
+  ) : null;
```

Comment:
```suggestion
  ) : (
    <Box className={classes.friendItem}>
      <Typography component="h3">
        <Skeleton />
      </Typography>
      <TextBody><Skeleton /></TextBody>
    </Box>
  );
```

Reply:
Does this mess up key stuff before friend is defined? Does it matter?

Reply:
I had this as `null` here since this is meant to represent the case when the friend is missing (due to some error in the backend data?) rather than a loading state, as this component is used with a map and doesn't get rendered until the query finishes loading.

I had this rendering something like "error loading user 5" or something before, but the catch is, the `userId` only exists when this is used with a friend request and not for the friend list. I didn't want to bloat this up and deal with the edge case so just went a bit sneaky and renders nothing for failed users instead.

Reply:
Actually, that might not be a problem anymore if I refactor the `useFriendList` hook to use `useUsers` instead and return the user map. In a hypothetical situation, if too many users are suddenly missing (probably unlikely though), the UI could look a bit noisy full of "error loading user X" messages so not sure that's ideal either.